modules/samtools/getrg/main.nf

```diff
@@ -0,0 +1,47 @@
+process SAMTOOLS_GETRG {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::samtools=1.15.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/samtools:1.15.1--h1170115_0' :
+        'quay.io/biocontainers/samtools:1.15.1--h1170115_0' }"
+
+    input:
+    tuple val(meta), path(input)
+
+    output:
+    tuple val(meta), file("readgroups.txt"),    emit: readgroup
```

Comment:
Perhaps you can read the content directly (untested)?

```suggestion
    tuple val(meta), file("readgroups.txt").text,    emit: readgroup
```

Reply:
Oooh, didn't know that was a thing! Let me try

Reply:
How would I make this compatible with pytest?

Reply:
Doesn't seem to work a first sight, I'm getting complaints the file doesn't exist.

Reply:
The output isn't a file right? maybe this can be changed to a value in `test.yml`?

Reply:
The error is from nextflow, not pytest
```console
         N E X T F L O W  ~  version 22.04.5                                                                                           
         Launching `./tests/modules/samtools/getrg/main.nf` [shrivelled_turing] DSL2 - revision: 8fef357436                            
         No such file: /Users/matdsmet/Projects/nf-core-modules/readgroups.txt                                                         
                                                                                                                                       
          -- Check script 'tests/modules/samtools/getrg/../../../../modules/samtools/getrg/main.nf' at line: 14 or see                 
         '.nextflow.log' file for more details                                                                                         
```

Reply:
maybe this will work? 
`tuple val(meta), val(file("readgroups.txt").text.strip()),    emit: readgroup`

Reply:
nope, same issue

Reply:
Sorry, then you can only do it after the process 🙁